### ISSUE TITLE
[openmc_libmesh] output shows up at the end of execution

### ISSUE THREAD

**Issue by AI-Pranto** [openmc-dev/openmc#1841]
Install Libmesh and OpenMC as following

```bash
../configure --prefix=$HOME/source_install/libmesh_install \
                  CXX=/usr/bin/mpicxx CC=/usr/bin/mpicc FC=/usr/bin/mpifort F77=/usr/bin/mpif77 \
                 --enable-exodus --enable-netcdf-4 --enable-eigen --enable-mpi --enable-hdf5 \
                 --disable-fortran --disable-lapack \
make -j 4 install

CC=/usr/bin/mpicc CXX=/usr/bin/mpicxx cmake ../openmc \
        -Dlibmesh=on -Ddebug=on \
        -DCMAKE_PREFIX_PATH=$HOME/source_install/libmesh_install
        -DCMAKE_INSTALL_PREFIX=$HOME/source_install/openmc_libmesh


make -j 4 install
```
And the output likes

![output](https://user-images.githubusercontent.com/49443884/120155592-d0032900-c212-11eb-9564-049567745e27.png)

- OS: Ubuntu 20.04. 

I was wondering what the problem is?

**Comment by gridley** [comment]
Libmesh overloads stuff in namespace std, cout included IIRC. @RonRahaman haven't you dealt with this before?

**Comment by aprilnovak** [comment]
I don't know for certain if this is the issue here, but libMesh has a [command line parameter](https://libmesh.github.io/doxygen/libmesh_8C_source.html#:~:text=%2F%2F%20redirect%20libmesh%3A%3Aout%20to%20nothing%20on%20all), `--keep-cout`, that if _not_ included on the command line, results in a lot of output being redirected to null.  

What happens if you pass `--keep-cout` on the command line?

**Comment by AI-Pranto** [comment]
@aprilnovak, Is `--keep-cout` command line argument? If I did, I got configure error `unrecognized option: `--keep-cout`

**Comment by RonRahaman** [comment]
Yes, by default, libmesh nullifies `std::cout` for all ranks except rank 0.  I raised a discussion here (same problem, different use case), but it hasn't been followed-up on.  

https://github.com/idaholab/moose/discussions/16240

**Comment by RonRahaman** [comment]
`libmesh::out` is nullified here.  Since `libmesh::cout` is a reference to `std::cout` in typical usage, it will affect OpenMC as well as libmesh

https://github.com/libMesh/libmesh/blob/9addd7b6e2e6e62aec0d47e883349368c79dfe91/src/base/libmesh.C#L622-L627
